generated/thrift/rpc.thrift

```diff
@@ -44,11 +44,13 @@ exception WriteBatchRawErrors {
 service Node {
 	// Friendly not highly performant read/write endpoints
 	FetchResult fetch(1: FetchRequest req) throws (1: Error err)
+	FetchTaggedResult fetchTagged(1: FetchTaggedRequest req) throws (1: Error err)
```

Comment:
Hm, `FetchTaggedResult` has an optional `FetchRawResult` field, so it should be `fetchTaggedRaw` I think.

Reply:
Ah nice catch. Migrated to using the uncompressed `datapoints` type in the response.

Reply:
Hm shouldn't we return raw data?  It seems we're using raw IDs (binary instead of string).

The "Friendly" endpoints all use string IDs and datapoints and the "Performant" endpoints all use raw compressed buffers and binary IDs.

Reply:
fair, next patch makes it 100% friendly.